_includes/tutorials/finding-distinct/ksql/markup/dev/print-topic.adoc

```diff
@@ -7,5 +7,5 @@ Finally, let's see what's available on the underlying Kafka topic for the table.
 The output should look similar to:
 
 +++++
-<pre class="snippet"><code class="shell">{% include_raw tutorials/finding-distinct/ksql/code/tutorial-steps/dev/outputs/print-topic/output-0.log %}</code></pre>
-+++++
\ No newline at end of file
+<pre class="snippet"><code class="shell">{% include_raw tutorials/finding-distinct/ksql/code/tutorial-steps/dev/expected-print-topic.log %}</code></pre>
```

Comment:
updated this to not look at output directory but expected outputs, as this is how it should be for the tutorial IMHO.

Reply:
Agree, we've done this in some others but I'm not sure if we've covered it across the board